src/components/SubMenuButton/SubMenuButton.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './SubMenuButton.css';
+
+function SubMenuButton({ name = 'Button', children, onClick }) {
+
+    return (
+        <button
+            className={`sub-menu-button sub-menu-button--${name}`}
+            onClick={onClick}
+        >
+            <span className={'sub-menu-button--text'}>{name}</span>
+            <span className={`sub-menu-button--icon sub-menu-button--icon-${name}`}>{children}</span>
+        </button>
+    );
+}
+
+export default SubMenuButton;
```

Comment:
it's not a submenu, it's just an element in menu.
So I would name it `MenuItem` or something like this